powerdns/serializers.py

```diff
@@ -42,6 +42,8 @@ class Meta:
 
 
 class RecordRequestSerializer(OwnerSerializer):
+
+    id = ReadOnlyField()
```

Comment:
do you really have to add this field by yourself? doesn't DRF handles it for you?